mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -2236,6 +2283,10 @@ impl UtxoRpcClientOps for ElectrumClient {
         )
     }
 
+    fn blockchain_scripthash_subscribe(&self, scripthash: String) -> UtxoRpcFut<Json> {
+        Box::new(rpc_func!(self, BLOCKCHAIN_SCRIPTHASH_SUB_ID, scripthash).map_to_mm_fut(UtxoRpcError::from))
+    }
```

Comment:
This subscribes with all electrums in the electrum list, but it's used when `TriggerSubscriptions` is sent below for each electrum https://github.com/KomodoPlatform/komodo-defi-framework/blob/681f0392ffdf6eceaf06bc9af8b32a6c221ca914/mm2src/coins/utxo/rpc_clients.rs#L1799-L1803
We can use `rpc_func_from` instead https://github.com/KomodoPlatform/komodo-defi-framework/blob/37e8a8df796c95de46d38055ef25dd057389c48a/mm2src/common/jsonrpc_client.rs#L23-L29
P.S. This is just nitpicking to reduce an additional request but resubscription won't cause a problem.

Reply:
Aren't we processing all Electrim client responses? If we are, it's very useful to subscribe to all of them to receive the fastest response possible. If not, what's the purpose of sending requests to all clients when we only care about one?

Reply:
Let's leave it as it is since we need to subscribe to all in `utxo_prepare_addresses_for_balance_stream_if_enabled`, what I meant is that we don't need to subscribe to all at once on `TriggerSubscriptions` since it will be triggered for each electrum when connected. Maybe a solution to both this and https://github.com/KomodoPlatform/komodo-defi-framework/pull/2013#discussion_r1414255782 is to use `TriggerSubscriptions` in `utxo_prepare_addresses_for_balance_stream_if_enabled` too, but this requires lots of changes since we subscribe to all addresses not only the one we added on `TriggerSubscriptions`. What do you think?
P.S. Like I said before this is just nitpicking to reduce overhead, we can keep it as is if you think it's better.

Reply:
I will resolve this since it's more related to how electrum connections and requests are managed and more related to this https://github.com/KomodoPlatform/komodo-defi-framework/pull/1966